packages/opentelemetry-node/src/propagator.ts

```diff
@@ -44,8 +53,31 @@ export class SentryPropagator implements TextMapPropagator {
   /**
    * @inheritDoc
    */
-  public extract(context: Context, _carrier: unknown, _getter: TextMapGetter): Context {
-    return context;
+  public extract(context: Context, carrier: unknown, getter: TextMapGetter): Context {
+    let newContext = context;
```

Comment:
l: Do we need `newContext` here, or can we just set directly on `context`?

Reply:
This was done because I didn't want to mutate the func param as per https://eslint.org/docs/latest/rules/no-param-reassign